plugins/woocommerce/src/Internal/Admin/Orders/MetaBoxes/CustomMetaBox.php

```diff
@@ -92,16 +92,51 @@ private function render_custom_meta_form( array $metadata_to_list, \WC_Order $or
 	 * Compute keys to display in autofill when adding new meta key entry in custom meta box.
 	 * Currently, returns empty keys, will be implemented after caching is merged.
 	 *
-	 * @param array|null         $keys Keys to display in autofill.
 	 * @param \WP_Post|\WC_Order $order Order object.
 	 *
 	 * @return array|mixed Array of keys to display in autofill.
 	 */
-	public function order_meta_keys_autofill( $keys, $order ) {
-		if ( is_a( $order, \WC_Order::class ) ) {
+	public function order_meta_keys_autofill( $order ) {
```

Comment:
Will changing the signature of this public method be problematic? I guess it's in the `Internal` namespace, so theoretically there shouldn't be any expectation of back compatibility. But still could be risky. What do you think?

Reply:
Addressed in 282453b5280088833cb693ad52d22497c9a5264d. Might be a bit ugly or unnecessary but it's backwards compatibility preserving 😄. Let me know what you think.

Reply:
I think this is fine. Naming the param `$deprecated` makes it very clear. Same thing WP Core does. 👍